example/diff-table.scss

```diff
@@ -52,6 +52,9 @@ $diff-bg-color-none-block-alternative: mix($diff-bg-color, $diff-table-sidebar-c
 
   th {
     font-weight: 700;
+    cursor: default;
+    -webkit-user-select: none;
```

Comment:
do we really need this?

Reply:
Sadly yes for safari

Reply:
Oh. That's my mistake then. Trying to work while on holiday is not as easy as it should be.

Reply:
We don't have to do this on your holiday :)